### PR TITLE
Add RSA mapping to NetStandard and NetCore crypto managers.

### PR DESCRIPTION
**Pull request by pmaytak**
Issue #2342

### CODE REVIEW

File: src/client/Microsoft.Identity.Client/Platforms/netstandard13/NetStandard13CryptographyManager.cs [hunk 1]
```diff
@@ -65,10 +69,18 @@ public byte[] Decrypt(byte[] encryptedMessage)
         /// <inheritdoc />
         public byte[] SignWithCertificate(string message, X509Certificate2 certificate)
         {
-            using (var key = certificate.GetRSAPrivateKey())
+            if (!s_certificateToRsaMap.TryGetValue(certificate.Thumbprint, out RSA rsa))
             {
-                return key.SignData(Encoding.UTF8.GetBytes(message), HashAlgorithmName.SHA256, RSASignaturePadding.Pkcs1);
+                if (s_certificateToRsaMap.Count >= s_maximumMapSize)
```
Comment (bgavrilMS) [inline src/client/Microsoft.Identity.Client/Platforms/netstandard13/NetStandard13CryptographyManager.cs:74]: It seems like there's a lot of code duplication between net461, netcore and netstandard. Perhaps move it in a single location.

**Review by bgavrilMS — APPROVED** [review]
LGTM, minor comment about consolidating duplicate code.

File: src/client/Microsoft.Identity.Client/Platforms/netstandard13/NetStandard13CryptographyManager.cs (hunk 1 above)
Comment (pmaytak) [reply src/client/Microsoft.Identity.Client/Platforms/netstandard13/NetStandard13CryptographyManager.cs]: @bgavrilMS I moved the code into CryptographyManager class. Does that look appropriate?

Comment (bgavrilMS) [reply src/client/Microsoft.Identity.Client/Platforms/netstandard13/NetStandard13CryptographyManager.cs]: LGTM

**Review by bgavrilMS — APPROVED** [review]

**Pull request merged**